### PR TITLE
display errors if happened

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: currently, all errors are digested and not displayed if happened

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/controller/service-account/service_account_binding_controller.go [hunk 1]
```diff
@@ -54,7 +55,11 @@ func (r *reconcileServiceAccountProjectBinding) Reconcile(request reconcile.Requ
 		return reconcile.Result{}, nil
 	}
 
-	return reconcile.Result{}, r.ensureServiceAccountProjectBinding(resourceName)
+	if err := r.ensureServiceAccountProjectBinding(resourceName); err != nil {
+		glog.Error(err)
```
Comment (p0lyn0mial) [inline api/pkg/controller/service-account/service_account_binding_controller.go]: we should at least add the name fo the controller to the error msg, there's more than one controller logging to the same "file", wdyt ?

Comment (zreigz) [reply api/pkg/controller/service-account/service_account_binding_controller.go]: yes, agree

**Comment by zreigz** [comment]
@p0lyn0mial PTAL

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

**Pull request merged**